### ISSUE TITLE
Issue with building with loop main branch / loop 3.4

### ISSUE THREAD

**Issue by dvayanu** [creepymonster/GlucoseDirectClient#6]
Hello, 
I was using Glucose Direct with previous loop directly. I assumed that with the 3.4 release of loop the client is necessary and should be used for the new main branch. 
I followed instructions in the video https://www.youtube.com/watch?v=7qWNzkWtUfQ except I took main branch instead of dev, and followed all other steps as well.
However, when building loop in xcode I get following errors (screenshot):

![Screenshot 2024-07-19 at 11 33 27](https://github.com/user-attachments/assets/76f90451-2c18-4be3-80a9-3c390f276ede)

Is there another way to get Loop 3.4 work with GlucoseDirect? 

thanks in advance
Leon

XCode 15.4, Apple M1, Sonoma, building for iphone 15 pro/ ios 17.0.1

Loop commit: 
Merge: 96772eb ba1b0d2
Author: Pete Schwamb <pete@schwamb.net>
Date:   Sat Jul 13 14:30:31 2024 -0500

    Merge pull request #151 from LoopKit/release-3.4
    
    * Updates for Release 3.4

**Comment by dvayanu** [comment]
Was able to solve it via browser build - https://www.loopandlearn.org/custom-code/

**Comment by FokkeZB** [comment]
@dvayanu mind sharing what steps you took to build it that way?

**Comment by dvayanu** [comment]
Hi @FokkeZB actually it was pretty straightforward once they updated the section.
1. https://loopkit.github.io/loopdocs/gh-actions/gh-overview/ followed the instructions here, with an active developer account this isn't a big deal. I already built previous versions, so I just updated my fork to 3.4
2. Downloaded GlucoseDirect and built with xCode, distributed it via TestFlight.
3. https://www.loopandlearn.org/custom-code/ <- goto the GCM part
![Screenshot 2024-07-30 at 12 24 15](https://github.com/user-attachments/assets/7bd0f4c4-fa2e-450c-b6b8-30526bf40a1c)
and follow the instructions. 
After editing the build_loop.yaml and building the new version had the GCM enabled.
here is my final build_loop file. 
https://github.com/dvayanu/LoopWorkspace/blob/main/.github/workflows/build_loop.yml

I struggled a little bit with indentation after c&p but in the end it all worked out.
